app/javascript/gobierto_visualizations/webapp/visualizations/treeMapNested.vue

```diff
@@ -273,11 +272,17 @@ export default {
         .paddingInner(0)
         .round(false);
 
+      const containerChart = d3.select(`.treemap-nested-container-${treemapId}`)
+
       const svg = d3.select(`#treemap-nested-${treemapId}`)
         .append("g")
         .attr('class', `treemap-container-${treemapId}`)
         .style("shape-rendering", "crispEdges");
 
+      containerChart.on('mouseleave', function() {
+        self.this.closeTooltips()
+      })
```

Comment:
May you simply something like that??? Asking
```suggestion
      containerChart.on('mouseleave', this.closeTooltips)
```

Reply:
Yeees!